backend/src/test/java/de/konstantintyker/audiophile/backend/artist/ArtistIntegrationTest.java

```diff
@@ -58,7 +55,7 @@ void deleteArtistByIdIsNotFound() throws Exception {
     @Test
     @DirtiesContext
     void deleteArtistWithExistId() throws Exception {
-        Artist savedArtist = testService.addNewArtist(new Artist("123", "Elton", "John"));
+        Artist savedArtist = testService.addNewArtist(new NewArtist("Elton", "John", "www.eltonjohn.com"));
         String id = savedArtist.id();
         mvc.perform(MockMvcRequestBuilders.delete("/api/artists/" + id))
```

Comment:
hier könntest du auch direkt die savedArtist.id als parameter in der URL benutzen.
`mvc.perform(MockMvcRequestBuilders.delete("/api/artists/" + savedARtist.id()))`

Reply:
somit würde die zwischenspeicherung wegfallen -- `String id = savedArtist.id()`